arq/worker.py

```diff
@@ -576,7 +576,10 @@ async def record_health(self) -> None:
             self._last_health_check_log = log_suffix
 
     def _add_signal_handler(self, signum: Signals, handler: Callable[[Signals], None]) -> None:
-        self.loop.add_signal_handler(signum, partial(handler, signum))
+        try:
+            self.loop.add_signal_handler(signum, partial(handler, signum))
+        except NotImplementedError:
+            logger.debug('Oy vey; Windows does not support adding a signal handler to an eventloop')
```

Comment:
```suggestion
            logger.debug('Windows does not support adding a signal handler to an eventloop')
```

Bit non standard and confusing.

Reply:
Solid. 

PS: windows regularly drives me bananas